Resources/Private/JavaScript/Host/Containers/Drawer/index.js

```diff
@@ -49,51 +110,46 @@ export default class Drawer extends Component {
     }
 
     renderMenu() {
-        const staticMenuData = [{
-            icon: 'file',
-            title: moduleLabel('content_menu_menuPanel_content'),
-            children: [{
-                icon: 'globe',
-                title: 'test'
-            }]
-        }, {
-            icon: 'briefcase',
-            title: moduleLabel('management_label', 'Modules'),
-            children: [{
-                icon: 'th-large',
-                title: 'Workspaces'
-            }, {
-                icon: 'camera',
-                title: 'Media'
-            }, {
-                icon: 'calendar',
-                title: 'History'
-            }]
-        }];
-
-        return staticMenuData.map((item, index) => this.renderMenuItem(item, index));
+        const {menuData} = this.props;
+
+        return Object.keys(menuData).map(k => menuData[k]).map((item, index) => this.renderMenuItem(item, index));
```

Comment:
While I think that every experienced developer knows that `k` is the iterated `key`, I would still prefer fully written variables names to be honest :smile: 

Nobody knows if we get some less experienced contributors on board to the project, and we should do those little things to make code reading a bit easier I guess, what do you think?